src/index.js

```diff
@@ -0,0 +1,145 @@
+let temperature = 48;
+
+const tempValue = document.getElementById('tempValue');
+const increaseTempControl = document.getElementById('increaseTempControl');
+const decreaseTempControl = document.getElementById('decreaseTempControl');
+const landScape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const cityNameReset = document.getElementById('cityNameReset');
+const headerCityName = document.getElementById('headerCityName');
+const getRealTimeTempButton = document.getElementById('getRealTimeTempButton');
```

Comment:
Looks like we are doing similar logic in a few places (finding an element by `id` and then, later in our code, adding an event handler to it for a specific event). Do you think that this could call for a helper function to make our code a bit more DRY?